src/man/sssd-secrets.5.xml

```diff
@@ -112,9 +112,9 @@ systemctl enable sssd-secrets.service
                     per-user subsections that define which provider store
                     the secrets for this particular user. The per-user
                     subsections should contain all options for that user's
-                    provider. If a per-user section does not exist, the
-                    global settings from the secret responder's section
-                    are used.  The following providers are supported:
+                    provider. Please note that currently the global provider
+                    is always local, the proxy provider can only be specified
+                    in a per-user section. The following providers are supported:
                     <variablelist>
```

Comment:
I think it is not clear what does it mean "per-user subsection"
We can have example here or reference somewhere else within man page